code/datums/components/wall_link.dm

```diff
@@ -0,0 +1,56 @@
+// This element should be applied to wall-mounted machines/structures, so that if the wall it's "hanging" from is broken or deconstructed, the wall-hung structure will deconstruct.
+/datum/component/wall_link
```

Comment:
Can we just name this wall_mounted?

Reply:
muh wall_mount element
![image](https://github.com/tgstation/tgstation/assets/58055496/02c87de3-de8f-40a4-98e0-ca8bb6e0a917)